### DISCUSSION TITLE
Contracting Party reference to multiple Buyers (Centralized purchasing scheme)

### DISCUSSION THREAD

**Discussion by DeanFirkelj** [OP-TED/eForms-SDK#527] [Q&A] [answered]
Hi, 
In case of CPB, when there are multiple organizations involved and listed in Organisations element (one of them having AwardingCPBIndicator -> true, and all other having AwardingCPBIndicator -> false), who should be exactly listed in ContractingParty element as reference? 

A. Only the CPB organization (the one having AwardingCPBIndicator -> true)
B. All of them as multiple Contracting Party elements - one as CPB and other as "buyers"

In case that B is the correct answer, the Contracting Party seems to be missing the Organization-Role element completely to distinguish the role itself. Actually, the values from the codelist Organisational Role cpb-awa or cpb-acq seem as they won't be used anywhere in contract notice.

**Comment by pdonohoe** [comment, marked as answer]
Hi,
The answer is closer to B. All buyer organizations involved should be listed in separate Organization elements. Where Awarding CPB is involved, at least one of these Organization elements must have AwardingCPBIndicator -> true (there can be more than one). Where Acquiring CPB is involved, at least one of these Organization elements must have AcquiringCPBIndicator -> true (there can be more than one). The Organization elements for all Buyers must be referenced from ContractingParty elements.

There is no need to use the codelist values cpb-awa or cpb-acq, as these roles are indicated by the AwardingCPBIndicator and AcquiringCPBIndicator elements. The information for each Buyer is a combination of the information in the ContractingParty element, and the Organization element(s) the ContractingParty links to, using Party/PartyIdentification/ID.

**Reply by JanaFafejtova** [reply]
Is it necessary to set the entire list of buyer organisations even in cases where several hundred/thousand entities are involved in centralised purchasing? Filling the entire list (including contact phone number and e-mail for each entity) is an extreme administrative burden for users. We believe, that in such cases it would be sufficient to fill in only one entity as the buyer which would be indicated as CPB.

**Reply by pdonohoe** [reply]
No it is not necessary. One organisation as a buyer, indicated as CPB would be sufficient. Whether and how many other organisations are also included as buyer organisation would be a decision for the main Buyer.

**Comment by DeanFirkelj** [comment]
Thnx. 
I suppose the identical principle is used in case of joint procurement scheme (having the attribute of GroupLeadIndicator = true or false)?

**Reply by pdonohoe** [reply]
Yes, except where GroupLeadIndicator is specified for one Buyer, it must be specified for all Buyers, and only one Buyer can have GroupLeadIndicator = true.
